apps/docs/src/common/pages/patterns/access-a-service.tsx

```diff
@@ -0,0 +1,119 @@
+import { FC, Fragment, createElement as h } from 'react';
+import { Helmet } from 'react-helmet-async';
+import { PageProps } from '@not-govuk/app-composer';
+import { A, NavigationMenu } from '@not-govuk/components';
+
+const applyAuthenticatePersonal = require('../../../../assets/images/p-authenticate-personal.svg').default;
```

Comment:
Why `p-authenticate-personal.svg` and not just `authenticate-personal.svg`?

If `p-` is for 'patterns' then you could put the images in an `assets/images/patterns/` directory.

Reply:
So I created a separate folder, linked my files to the new path and then something broke! I kept the links with "p-" for now to figure where I went wrong  but can investigate later.

Reply:
You might need to restart the app.